staging/src/k8s.io/cli-runtime/pkg/resource/helper.go

```diff
@@ -38,6 +38,13 @@ type Helper struct {
 	RESTClient RESTClient
 	// True if the resource type is scoped to namespaces
 	NamespaceScoped bool
+	// If true, then use server-side dry-run to not persist changes to storage
+	// for verbs and resources that support server-side dry-run.
+	//
+	// Note this should only be used against an apiserver with dry-run enabled,
+	// and on resources that support dry-run. If the apiserver or the resource
+	// does not support dry-run, then the change will be persisted to storage.
+	DryRunServer bool
```

Comment:
Would ServerSideDryRun be better name for the field ?
Normally I wouldn't expect something ending in Server to be of bool type.

Reply:
I don't hate "ServerDryRun", but I don't think it makes a huge difference? Thanks

Reply:
Renamed!